.github/ISSUE_TEMPLATE/3_technical_task.md

```diff
@@ -3,7 +3,13 @@ name: 🔨 Technical task
 about: Task that is not necessarily user facing
 ---
 
-<!-- A clear and concise description of what needs to be done. -->
+<!-- A clear and concise description of what needs to be done.
+
+Include a Definition of Done: acceptance criteria and examples for this task.
```

Comment:
The former should be checkboxes. Maybe 'Tasks' needs expanding?

Reply:
If it includes various example, definitely checkboxes. There are more cases in which that would be needed e.g. anything involving multiple repositories and PRs. Can see sometimes could be just a phrase for a small ticket.

Reply:
But, it's different from the `Tasks` as it's the what rather than the how.